docs/production/mobile-push-notifications.md

```diff
@@ -32,6 +32,8 @@ follows:
     su zulip -c '/home/zulip/deployments/current/manage.py register_server'
     # Or as the zulip user, you can skip the `su zulip -c`:
     /home/zulip/deployments/current/manage.py register_server
+    # On docker:
+    docker exec -it -u zulip <container name> /home/zulip/deployments/current/manage.py register_server
```

Comment:
Do we expect the container name to be `zulip`?

Reply:
The `-u` flag denotes user impersonation. Actually this got me at first so I had to run the command again. You cannot run this command in the docker container as root (or any other user), it has to be run as the user `zulip`. This is required.

The part where it says `<container name>` needs to be replaced by the user running this command because I don't know what container name the user will have chosen. By default it is `docker-zulip_zulip_1` if you cloned it from the repo.

Obviously none of this is intuitive and requires you to either have all this memorized by heart or you have to alt-tab a few times to gather the missing pieces of information so I thought I'd reduce a bit of work for whoever's reading this in the future.